scripts/remove-unused-keys.js

```
@@ -0,0 +1,156 @@
+      (key, options) => {
+        parser.set(key, options);
+        allUsedKeys.add(key);
+      },
+    );
+
+    // To handle Trans component, transform the file into js and scan it for Trans component
+    if (
+      content.includes("Trans") &&
+      (content.includes("react-i18next") || content.includes("i18next"))
+    ) {
+      let content = content;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)